curvefs/conf/mds.conf

```diff
@@ -64,3 +64,25 @@ mds.heartbeat.intervalMs=10000
 mds.heartbeat.misstimeoutMs=30000
 # heartbeat offline time between metaserver and mds
 mds.heartbeat.offlinetimeoutMs=1800000
+# After mds is started, a certain time delay starts to guide the metaserver to delete data
+mds.heartbeat.clean_follower_afterMs=1200000
```

Comment:
// After mds is started, a certain time delay starts to guide the metaserver to delete data
// default is 20min

Reply:
done